README.md

```diff
@@ -24,5 +24,5 @@ composer require yiisoft/widget
 
 The package is tested with PHPUnit. Tests could be run with
 
-./vendor/bin/phpunit
+`./vendor/bin/phpunit`
```

Comment:
```suggestion
`composer test`
```
I suggest adding the ability to run tests through Comopser
To do this, edit the `composer.json` file as [here](https://github.com/yiisoft/injector/blob/b715872f699ec4f521b39039bf9f6c4e253cfa52/composer.json#L44)

Reply:
I've updated PR, but I didn't add `--colors=always`, because it configured in `phpunit.xml.dist` file